cachito/web/models.py

```diff
@@ -1,11 +1,69 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
+from copy import deepcopy
 
 from cachito.web import db
 
 
+request_pkg_manager_table = db.Table(
+    'request_pkg_manager',
+    db.Model.metadata,
+    db.Column('request_id', db.Integer, db.ForeignKey('request.id')),
+    db.Column('pkg_manager_id', db.Integer, db.ForeignKey('pkg_manager.id'))
+)
+
+
 class Request(db.Model):
     """A Cachito user request."""
+    __tablename__ = 'request'
     id = db.Column(db.Integer, primary_key=True)
+    git_repo = db.Column(db.String, nullable=False)
+    git_ref = db.Column(db.String, nullable=False)
+    pkg_managers = db.relationship('PackageManager', secondary=request_pkg_manager_table)
```

Comment:
It might be nice to use `backref` here so that from a package manager, we can get all the requests.

Reply:
Good call! Done.